hazelcast-jclouds/src/main/java/com/hazelcast/jclouds/JCloudsProperties.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jclouds;
+
+import com.hazelcast.config.properties.PropertyDefinition;
+import com.hazelcast.config.properties.PropertyTypeConverter;
+import com.hazelcast.config.properties.SimplePropertyDefinition;
+import com.hazelcast.config.properties.ValueValidator;
+
+import static com.hazelcast.config.properties.PropertyTypeConverter.INTEGER;
+import static com.hazelcast.config.properties.PropertyTypeConverter.STRING;
+
+/**
+ * The type J clouds properties.
+ */
+public final class JCloudsProperties {
+    /**
+     * Unique identifier for ComputeService Provider
+     * see the full list of ids : https://jclouds.apache.org/reference/providers/
+     */
+    public static final PropertyDefinition PROVIDER = property("provider", STRING);
+
+    /**
+     * Unique credential identity specific to users cloud account
+     */
+    public static final PropertyDefinition IDENTITY = property("identity", STRING);
+    /**
+     * Unique credential specific to users cloud accounts identity
+     */
+    public static final PropertyDefinition CREDENTIAL = property("credential", STRING);
+    /**
+     * Property used to define zones for node filtering
+     */
+    public static final PropertyDefinition ZONES = property("zones", STRING);
+    /**
+     * Property used to define regions for node filtering
+     */
+    public static final PropertyDefinition REGIONS = property("regions", STRING);
+    /**
+     * Property used to define tag keys for node filtering
+     */
+    public static final PropertyDefinition TAG_KEYS = property("tag-key", STRING);
+    /**
+     * Property used to define tag values for node filtering
+     */
+    public static final PropertyDefinition TAG_VALUES = property("tag-value", STRING);
```

Comment:
tag_keys, tag_values can be comma separated. I think this could be mentioned in java doc.